website/docs/components/breadcrumb/partials/code/how-to-use.md

```diff
@@ -19,9 +19,9 @@ In the example above, we have omitted a few parameters for clarity. In reality,
 
 ```handlebars
 <Hds::Breadcrumb>
-  <Hds::Breadcrumb::Item @text="My org" @icon="org" @route="index" />
-  <Hds::Breadcrumb::Item @text="Consul" @icon="consul" @route="list" />
-  <Hds::Breadcrumb::Item @text="my-consul-cluster" @route="detail.index" @model={{@cluster.id}} />
+  <Hds::Breadcrumb::Item @text="My org" @icon="org" @route="components" />
```

Comment:
Do you mind leaving this out? I know it breaks this page, but I may have a hack/solution (introduce a custom/ghost set of routes `my.page.something.something` for all the documentation pages).

Reason being, when there are code snippets that are used as examples, in some cases is better to add some form of semantic to the route name, to convey a meaning (in this case, the hierarchy of the routes)

Reply:
Actually, rethinking of this: let's push the fix, and we will change once we have the `my.page` routes hack in place.

Reply:
The example following this one was using `@route="components"` in all 'crumbs' so I thought this would also align them – a hack for `@route`s would be great as we have many examples making use of them.